draft-ietf-taps-interface.md

```diff
@@ -1427,6 +1427,10 @@ temporary local addresses might involve trading off privacy for performance.
 For instance, temporary addresses (e.g., {{?RFC8981}}) can interfere with resumption mechanisms
 that some protocols rely on to reduce initial latency.
 
+The default is Prefer for Rendezvous since the expectation is that peer-to-peer
+connections are short-lived. Listeners, that are expected to be long-lived public
+services default to avoiding temporary local addresses.
```

Comment:
If Listeners and Rendezvous are mentioned explicitly, should Initiate also be mentioned? 
I thinkit is the only one left.

Reply:
IMO the default for Initiate is probably a matter of system policy, so I'm fine with the text as is.